service/models.py

```diff
@@ -0,0 +1,90 @@
+import logging
+from flask import Flask
+from flask_mongoengine import MongoEngine
+from flask_mongoengine.wtf import model_form
+
+db = MongoEngine()
```

Comment:
I think **MongoEngine()** should accept a parameter **app** indicating which DB to connect?